utils/resolve.js

```diff
@@ -10,7 +10,8 @@ const path = require('path');
 const hashObject = require('./hash').hashObject;
 const ModuleCache = require('./ModuleCache').default;
 
-const CASE_SENSITIVE_FS = !fs.existsSync(path.join(__dirname.toUpperCase(), 'reSOLVE.js'));
+const projectDir = pkgDir.sync(__dirname);
```

Comment:
is this basically just avoiding hardcoding `./package.json`?

Reply:
Oh, whoops -- I thought `pkg-dir` skipped the current directory and started looking for a `package.json` in the parent directory.

So the logic should be trying to read `/Users/sindresorhus/foo/package.json`, not `/Users/sindresorhus/foo/node_modules/eslint-module-utils/package.json`.

```
/
└── Users
    └── sindresorhus
        └── foo
            ├── package.json
            └── node_modules
                ├── eslint-module-utils
                └── package.json
                └── resolve.js
```

Reply:
In that case, it'd be `path.join(process.cwd(), 'package.json')` - but also, that wouldn't work for a project that doesn't have a package.json (altho that's a rare use case)